torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1992,14 +1994,19 @@ void ONNXShapeTypeInference(
       //       The conversion here is incomplete for these ops.
       //       e.g: ListConstruct, ListUnpack, etc.
       std::shared_ptr<onnx::ModelProto> model_proto;
-      SymbolDimMap symbol_map;
       ConvertGraphToONNXProto(n_graph, model_proto, symbol_map, opset_version);
       GRAPH_DEBUG(
           "ONNX graph to run shape inference: ", prettyPrint(*model_proto));
 
       // infer shape
       try {
-        onnx::shape_inference::InferShapes(*model_proto);
+        // Now here only supports Shape and Gather
```

Comment:
this comment is redundant with the code immediately below it, so it adds no value. Just remove this line.